qiskit/synthesis/evolution/suzuki_trotter.py

```diff
@@ -56,6 +55,18 @@ def __init__(
             Callable[[Union[Pauli, SparsePauliOp], float], QuantumCircuit]
         ] = None,
     ) -> None:
+        """
```

Comment:
For some reason, yes - I think it might be a Sphinx parsing bug.  If I let Sphinx try to inherit the `__init__` docstring from the base class, it complains about invalid indentation (perhaps it doesn't join the indentation levels together correctly or something).